### PR TITLE
test(integration): Add snapshot tests

### PR DESCRIPTION
**Pull request by erezrokah**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

Trying a new testing approach to catch issues like https://github.com/cloudquery/cloudquery/pull/830

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/.github/CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests 🧪
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

File: test_integration/.snapshots/TestIntegrationCommands-policy-output [hunk 1]
```diff
@@ -0,0 +1,96 @@
+  STATUS | REASON | ACCOUNT ID   | ARN                            | REGION
```
[the remark this answers is not in the document]
Comment (erezrokah) [reply test_integration/.snapshots/TestIntegrationCommands-policy-output:1]: Should be fixed in https://github.com/cloudquery/cloudquery/pull/853

File: test_integration/commands_test.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+var testCases = map[string]testCase{
+	"policy-output": {
+		setupCommands: [][]string{
+			{"dropdb", "-h", dbHost, "-p", dbPort, "-U", dbUser, dbName},
+			{"createdb", "-h", dbHost, "-p", dbPort, "-U", dbUser, dbName},
+			// Created using `pg_dump -h 127.0.0.1 -U postgres test_integration > test_integration/fixtures/fetch_data/aws_s3.pgsql` after fetching only `s3*` resources, and sanitizing the output
+			{"psql", fmt.Sprintf("host=%s port=%s dbname=%s user=%s", dbHost, dbPort, dbName, dbUser), "-f", getFixtureFilePath("fetch_data/aws_s3.pgsql")},
+		},
+		actCommand: []string{"go", "run", mainFile, "policy", "run", "github.com/cloudquery-policies/aws?ref=v0.1.14", "--config", getFixtureFilePath("fetch_data/aws_config.hcl")},
```
Comment (erezrokah) [inline test_integration/commands_test.go:41]: Using pinned versions of the provider (in `config.hcl`) and policy (in the argument) to keep the test stable, as we want to test only the output part

Comment (erezrokah) [inline test_integration/commands_test.go:41]: Using pinned versions of the provider (in `config.hcl`) and policy (in the argument) to keep the test stable, as we want to test only the output part

**Comment by yevgenypats** [comment]
I would hold off with that until we re-write the policies to yaml and remove a lot of complexities including go gitter and so on, otherwise we will have more code to re-write in one go.

**Comment by erezrokah** [comment]
Going to close this for now

**Pull request closed without merging**